lib/perl/Genome/Model/Build/SomaticVariation.pm

```diff
@@ -484,4 +484,29 @@ sub build_for_tumor_bam {
     return $self->tumor_build;
 }
 
+sub _disk_usage_result_subclass_names {
+    my $self = shift;
+
+    my @disk_usage_result_classes = (
+        'Genome::InstrumentData::AlignmentResult::BwamemStream',
+        'Genome::InstrumentData::AlignmentResult::Merged',
+        'Genome::InstrumentData::AlignmentResult::Merged::BamQc',
+        'Genome::Model::Tools::DetectVariants2::Classify::Loh',
+        'Genome::Model::Tools::DetectVariants2::Classify::PreviouslyDiscovered',
+        'Genome::Model::Tools::DetectVariants2::Classify::Tier',
+        'Genome::Model::Tools::DetectVariants2::Result',
+        'Genome::Model::Tools::DetectVariants2::Result::Combine::IntersectSnv',
+        'Genome::Model::Tools::DetectVariants2::Result::Combine::LqUnion',
+        'Genome::Model::Tools::DetectVariants2::Result::Combine::UnionSv',
```

Comment:
I see this in the list of classes skipped:
`Genome::Model::Tools::DetectVariants2::Result::Combine::UnionSv`